tests/unit/article.test.js

```diff
@@ -0,0 +1,53 @@
+import 'chai/register-should';
+import { expect } from 'chai';
+import { validateArticle } from '../../src/utils/article';
+
+const ARTICLE = {
+  title: 'Andela is cool 7888',
+  userId: '979eaa2e-5b8f-4103-8192-4639afae2ba8',
+  description: 'Lorem ipsum dolor sit amet, sit ut dolor alterum, sed malis referrentur cu. Aperiam fabulas eos ea. Sea mazim senserit tincidunt te.',
+  body: 'Lorem ipsum dolor sit amet, sit ut dolor alterum, sed malis referrentur cu. Aperiam fabulas eos ea. Sea mazim senserit tincidunt te. Mei volutpat delicatissimi ut, id mollis alienum argumentum has, semper efficiendi sed ea. Ius decore consul forensibus ne, enim verear corpora sit ut. Usu eu possit equidem menandri, quo et noster officiis iracundia.',
+  imageUrl: 'https://picsum.photos/200/300',
+  tags: ['hello', 'async', 'await']
+};
+describe('validateArticle()', () => {
+  it('should return true if the validation passes', async (done) => {
```

Comment:
Why are you using async with thenables? Also, why the done callback?

Reply:
@Temmyogunbo Will implement your feedback